scripts/run-gtest.ps1

```diff
@@ -97,7 +97,7 @@ param (
     [switch]$BreakOnFailure = $false,
 
     [Parameter(Mandatory = $false)]
-    [ValidateSet("None", "Basic.Light", "Basic.Verbose", "Full.Light", "Full.Verbose")]
+    [ValidateSet("None", "Basic.Light", "Datapath.Light", "Datapath.Verbose", "Stacks.Light", "Stacks.Verbose", "RPS.Light", "RPS.Verbose", "Performance.Light", "Basic.Verbose", "Performance.Light", "Performance.Verbose", "Full.Light", "Full.Verbose", "SpinQuic.Light", "SpinQuicWarnings.Light")]
```

Comment:
I was in the situation where I wanted to take a Stacks trace during a functional test case and this seemed like a useful change. I can revert this part if desired though